pkg/filelist.ml

```diff
@@ -139,6 +139,12 @@ let ocamlbuild = {
 
 }
 
+let ppx = {
+  interface = [ "ppx_eliom" ; "ppx_eliom_client" ; "ppx_eliom_type" ; "ppx_eliom_server" ];
+  internal = [ "ppx_eliom_utils" ];
+}
```

Comment:
Add an `interface_only` field here. This is for `mli`-only modules, which have started appearing in Eliom. This fixes the build failure